.github/workflows/daily.yml

```diff
@@ -21,16 +21,46 @@ jobs:
     strategy:
       matrix:
         block:
+          - 700000
+          - 705000
+          - 710000
+          - 715000
+          - 720000
+          - 725000
+          - 730000
+          - 735000
           - 740000
-          - 741000
-          - 742000
-          - 743000
-          - 744000
           - 745000
-          - 746000
-          - 747000
-          - 748000
-          - 749000
```

Comment:
Why replacing these blocks?

Reply:
Since we already know those blocks are working properly, I changed them to get a little extra coverage

Reply:
I added back those blocks to compare with previous execution